lib/mayaUsd/resources/scripts/mayaUsdUtils.py

```diff
@@ -193,4 +193,39 @@ def saveWantPayloadLoaded(want):
     opVarName = "mayaUsd_WantPayloadLoaded"
     cmds.optionVar(iv=(opVarName, want))
 
+def showHelpMayaUSD(contentId):
+    """
+    Helper method to display help content.
+
+    Note that this is a wrapper around Maya's showHelp() method and showHelpMayaUSD()
+    should be used for all help contents in Maya USD.
+
+    Example usage of this method:
+    
+    - In Python scripts:
+    from mayaUsdUtils import showHelpMayaUSD
+    showHelpMayaUSD("someContentId");
+
+    - In MEL scripts:
+    python(\"from mayaUsdUtils import showHelpMayaUSD; showHelpMayaUSD('someContentId');\")
+    
+    - In C++:
+    MGlobal::executePythonCommand(
+    "from mayaUsdUtils import showHelpMayaUSD; showHelpMayaUSD(\"someContentId\");");
+
+    Input contentId refers to the contentId that is registered in helpTableMayaUSD
+    file which is used to open help pages.
+    """
+    import os
     
+    try:
+        # Finding the path to helpTableMayaUSD file.
+        dirName = cmds.pluginInfo('mayaUsdPlugin', q=True, p=True)
+        dirName = os.path.dirname(dirName) + "/../../../lib/helpTable/" + "helpTableMayaUSD"
```

Comment:
Sorry this is my fault now. Since this is now in mayausd lib I don't think we should use the path of the mayaUsdPlugin. Instead I would suggest adding the line
`MAYAUSD_LIB_LOCATION:=lib` 
in the `MayaUSD_LIB` section of the mayaUSD.mod file (don't forget to add to the windows one too). I'm actually surprised we don't already have a variable for this path. You'll see we already have one for `USD_LOCATION`.

Then here you can get the path to the helpTable with:
`helpTablePath = os.path.join(os.environ['MAYAUSD_LIB_LOCATION'], 'helpTableMayaUSD')`